### ISSUE TITLE
Autoplay on mobile devices

### ISSUE THREAD

**Issue by Eelsie** [vimeo/player.js#257]
### Expected Behavior

Videos autoplay on mobile devices.

### Actual Behavior

Videos autoplay on desktop, but not on mobile.

### Question

Is it possible to autoplay videos on android and IOS devices? I'm actually using these parameters `src="`https://player.vimeo.com/video/${vimeoId}?autoplay=1&loop=1&autopause=0&muted=1&background=1&playsinline=1`"` and I also tried to trigger the play on load with the player `player.play()`. Thanks for your help!

**Comment by rsichel107** [comment]
@Eelsie I have tested your link with this video (`76979871`) at https://player.vimeo.com/video/76979871?autoplay=1&loop=1&autopause=0&muted=1&background=1&playsinline=1 on Android 8.1 with Chrome 66 (a Pixel 2) and it autoplayed perfectly. Also was tested on and iPhone 7 running the newest software 11.3.1 and it also autoplayed. 

It is worth noting that you have many excessive parameters. 
`background=1` is equivalent to `autoplay=1&loop=1&muted=1' as background will always, autoplay, muted and loop. So your link should really be: `https://player.vimeo.com/video/76979871?background=1&autopause=0&playsinline=1`

If you provide more details, perhaps we can help. Otherwise, I will close this ticket.

**Comment by rsichel107** [comment]
It is also worth noting that Autoplay does not work on iPhones in power save mode or Androids in data save mode so that may be part of your problem.

**Comment by pxwee5** [comment]
Tried this and failed on Android 8 Chrome. 
But plays on iOS Safari. 

Suspects that you guys may have accumulated enough Chrome MEI score to play from codepen. Currently there are no ways of resetting Chrome MEI scores.

**Comment by rsichel107** [comment]
@pxwee5 I have found on my Pixel 2, Android 8 in Chrome it does not autoplay when I am turn on "Data Saver" In Chrome Settings -> Data Saver. When I turn that off it does.  
To test: please go to https://player.vimeo.com/video/76979871?autoplay=1&muted=1 on your phone and ensure Data Saver is off. Does it autoplay?

**Comment by pxwee5** [comment]
@rsichel107 Thanks for replying, but unfortunately no, it does not autoplay on my phone. 
I'm on charging (no power save mode) and wifi (no data saver).

**Comment by rsichel107** [comment]
@pxwee5 Sorry if that was unclear, data saver applies whether you are on Wi-fi or not and you may not even realize it is on for you. As you can see from my screenshot I am also on Wi-Fi but datasaver is still on. Again, please confirm that the settting is not on. 
![screenshot_20180712-113824](https://user-images.githubusercontent.com/17985323/42644652-1d1783a4-85ca-11e8-8ac5-1494038adf90.png)

**Comment by pxwee5** [comment]
@rsichel107 Nope, it's not on.

**Comment by rsichel107** [comment]
@pxwee5 I'm sorry I can't be more helpful, this works on our Android test devices.

**Comment by maximilliangeorge** [comment]
Would it be possible to fall back to ordinary player when "low power mode" and "data saving mode" is enabled? Or some other fallback option?

**Comment by bendytree** [comment]
If anyone's having trouble with autoplay within a `WKWebView` on iOS, then make sure you set `allowsInlineMediaPlayback` to `true`. It defaults to `false`.

**Comment by Quberik** [comment]
Yes, it is possible. But I don't know how to do that. I saw an example here http://greencow.ru

**Comment by j-greig** [comment]
I don't see `allowsInlineMediaPlayback` mentioned on https://vimeo.zendesk.com/hc/en-us/articles/360001494447-Using-Player-Parameters anywhere, can it be used as a URL parameter?
